### PR TITLE
Fixed main page CSS

### PR DESCRIPTION
**Pull request by peledni**
Related to: https://github.com/ClickHouse/product/issues/40

### CODE REVIEW

File: src/css/custom.css [hunk 1]
```diff
@@ -265,14 +265,32 @@ code {
...
   }
 }
+
+#__docusaurus {
+    height: 100%;
+}
+
+.navbar {
+    flex: 0 0 var(--ifm-navbar-height);
+}
```
Comment (gingerwizard) [inline src/css/custom.css:296]: insert empty line

**Review by gingerwizard — CHANGES_REQUESTED** [review]

**Review by rfraposa — APPROVED** [review]
I don't know what it does, but curious to see if it works. Thanks for your help!

**Comment by rfraposa** [comment]
The home page looks good, but the footer is getting pushed down beyond the bottom on all the other pages. I will leave it merged for now...and see if that's an issue with anyone

**Comment by peledni** [comment]
@rfraposa I did that on purpose, pages without a lot of content looked a bit bad (and I checked other sites using docsaurus and they do the same). If you want to revert it it's just removing 1 line of css.

**Comment by rfraposa** [comment]
@peledni  - it's growing on me! Let's leave it 😄

**Pull request merged**